style.css

```diff
@@ -432,15 +431,13 @@ ul {
 }
 
 .allabout {
-  width: 70%;
-  flex-basis: 37rem;
+  width: 47%;
 }
```

Comment:
- [ ] Again, we are going to put our focus on the about myself section. If you notice, the desktop screen size at 908px. The `.knowlegdes` overlaps this `.allabout` 😞 . Since we need to stick to the Figma template design as much as possible. I kindly suggest the spacing(margin, padding, etc) matches the Figma design to help us have a professional portfolio ☺️ .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/222513822-910e8ab3-9e47-406c-bf89-922eed06ca03.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/222513904-0742ec9d-778e-4e39-9376-3de1dfa24693.png)